state:

  # Upper and lower bounds for experimental space (meters). [x, y, z]
  lower: [-2.0, -1.04, 0] 
  upper: [1.66, 2.62, 2.0] 

pred:

  # Start position of human(s) and list of potential goal locations
  # linear sim start: [1.63, 0.46]
  # linear sim goals: [[-1.44, 0.44]]
  # coffee_real_start: [1.61,-1.02]
  # coffee_real_goals: [[-0.17, 2.60]]
  human_real_start: [1.63, 0.46] #[1.61,-1.02]
  human_real_goals: [[-1.44, 0.44]] #[[-0.17, 2.60]]

  human1_real_start: [1.63, 0.46] #[1.61,-1.02] 
  human1_real_goals: [[-1.44, 0.44]] #[[-0.17, 2.60]]

  # color coding for human visualization (RGB)
  human_color: [1.0, 0.0, 0.0]

  human1_color: [1.0, 0.0, 0.0]

  # Dimensions of the prediction map
  sim_height: 26
  sim_width: 26

  # Resolution, ex. (real meters)/(sim dim-1) (m/cell)
  resolution: 0.1464 

  # Num timesteps into the future to predict
  fwd_tsteps: 10

  # Initial betas depending on model
  beta_adaptive: [0.1, 0.3, 1.0, 3.0, 10.0]
  beta_rational: [0.05]
  beta_irrational: [10.0]

  # Human's height (in meters)
  human_height: 1.67

  # Human's walking speed (m/s) -- human walking speed is 1.4 m/s
  human_vel: 0.7 

  # Probability threshold 
  prob_thresh: 0.001

  # Process noise (epsilon-stubborn model)
  epsilon_dest: 0.02
  epsilon_beta: 0.02


